tests/output/InteractionsTest.py

```diff
@@ -12,11 +13,33 @@ class InteractionsTest(unittest.TestCase):
     def test_(self):
         log_printer = LogPrinter(NullPrinter())
         section = Section('')
-        self.assertRaises(TypeError, fail_acquire_settings, log_printer, None,
-                          section)
-        self.assertRaises(AssertionError,
-                          fail_acquire_settings,
-                          log_printer,
-                          {'setting': ['description', 'bear']}, section)
-        self.assertEqual(fail_acquire_settings(log_printer, {}, section), None,
-                         section)
+
+        with LogCapture() as capture:
+
+            self.assertRaises(TypeError, fail_acquire_settings, log_printer,
+                              None, section)
+
+            self.assertRaises(AssertionError,
+                              fail_acquire_settings,
+                              log_printer,
+                              {'setting': ['description', 'bear']}, section)
+
+            self.assertEqual(fail_acquire_settings(log_printer, {}, None),
+                             None, section)
+
+            capture.check(
+              ('root',
+               'WARNING',
+               'fail_acquire_settings: section parameter'
+               ' is deprecated.'),
+              ('root',
+               'WARNING',
+               'fail_acquire_settings: section parameter'
+               ' is deprecated.'),
+              ('root',
+               'ERROR',
+               'During execution, we found that some required settings were'
+               ' not provided. '
+               'They are:\n'
+               'setting (from bear) - description')
+            )
```

Comment:
I think you should keep the test case as is, except removing `section` from the function calls. Then you introduce a new test function checking the deprecation message.

Reply:
bump ;)